docs/docs/build/examples/btc-swap/index.mdx

```diff
@@ -0,0 +1,426 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import SellOrdinalConnectedImageUrl from "./add-sell-ordinal-connected-order-form.png";
+import SellOrdinalDisconnectedImageUrl from "./add-sell-ordinal-disconnected-order-form.png";
+
+# Trustless P2P Swap BTC and ERC20 using a BTC Light Client
+
+This example demonstrates how BOB is leveraged to build a peer-to-peer (P2P) swap application that allows two parties to swap ERC20 tokens, BTC and Ordinals without using a custodian.
+
+![demo-overview](demo-overview.png)
+
+:::info Example Code
+
+You can find the code for this example [in the P2P swap demo GitHub repository](https://github.com/bob-collective/demo-p2p-swap/releases/tag/v0.0.1).
+
+:::
+
+## Demo
+
+:::info Example Code
+
+This demo is running on our [Alpha network](https://app.conduit.xyz/published/view/fluffy-bob-7mjgi9pmtg).
+
+:::
+
+We have created a testnet demo of the P2P swap, allowing anyone to swap BTC for ERC20 tokens, ERC20 tokens for BTC, exchange one ERC20 token for another and swap Ordinals for ERC20 tokens. The application currently supports testnet BTC, testnet USDC, and WBTC (WBTC is a placeholder for an arbitrary bridged BTC token).
+
+## Connecting Dual Wallets
+
+The demo supports connecting dual wallets, which means that you should be able to connect an EVM wallet and a BTC wallet at the same time.
+
+- EVM Wallet: it is mandatory to install this type of wallet. Without it you won't be able to try the demo.
+- BTC Wallet: it is not mandatory, but the UX experience will drastically improve when it comes to swapping ordinals.
+
+### Connecting EVM Wallet
+
+:::note
+
+When connecting to our test network the gas token ticker may be shown as something other than ETH. This can be changed to ETH in the network settings of your wallet.
+
+:::
+
+:::note
+
+The UI can sometimes fail to connect if you have multiple wallets installed. If this happens you should disable all wallet extensions other than the one you want to use for the demo.
+
+:::
+
+- Install one of the supported EVM wallets (Metamask, Ledger, WalletConnect or Coinbase Wallet).
+- Go to [demo.gobob.xyz](https://demo.gobob.xyz)
+- Click 'Connect Wallet' button at the top right of the UI.
+- Within the 'EVM Wallet' tab, select one of the available wallets.
+
+![connect-evm](connect-evm-wallet.png)
+
+### Connecting BTC Wallet
+
+- Install one of the supported BTC wallets (XVerse, Unisat, Leather - coming soon).
+- Go to [demo.gobob.xyz](https://demo.gobob.xyz)
+- Click 'Connect Wallet' button at the top right of the UI.
+- Within the 'BTC Wallet' tab, select one of the available wallets.
+
+![connect-btc](connect-btc-wallet.png)
+
+## Creating and Funding Test Accounts
+
+### Funding EVM Wallet
+
+You will need two accounts for this demo, so you can try the whole flow of creating and fulfilling orders. For each account:
+
+- Click the 'Get Gas' button to fund your account with ETH. This will be needed for transaction fees.
```

Comment:
Update this. You can also refer to this guide here to get ETH: https://docs.gobob.xyz/docs/learn/guides/#bob-testnet